### DISCUSSION TITLE
Housing data

### DISCUSSION THREAD

**Discussion by jacobrharris** [datakind/datakit-housing-fall-2024#54] [Announcements]
This release addresses a bug that caused incomplete data for Florida. The updated `data_1-FL.csv` file now contains all rows for all Florida counties. A fix is in progress for a similar issue with California data and is not included in this release. There are no changes to the California data just yet.

<hr /><em>This discussion was created from the release <a href='https://github.com/datakind/datakit-housing-fall-2024/releases/tag/v1.1'>Housing data</a>.</em>

**Comment by MELSHAMA** [comment]
It seems to me that the column names are renamed in a way that we no longer able to use the data dictionary. If you find this true at your side, I suggest to provide the dataset before renaming that is consistent with the data_dictionary file because it allows us to access to the source of data when needed and/or the rename map.

**Reply by MELSHAMA** [reply]
Please ignore my comment above. The data_dictionary and the dataset seem consistent. Sorry for the inconvenience.
